Shared/Freak.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+struct Freak {
+    var imageName: String
```

Comment:
We should use `let` instead of `var`. As a rule of thumb, use `var` only where is really necessary, otherwise by default use let.